community/neo4j/src/test/java/migration/Start3_2DbOn3_3AndCreateFusionIndexIT.java

```diff
@@ -0,0 +1,208 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package migration;
+
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.factory.GraphDatabaseFactory;
+import org.neo4j.graphdb.schema.IndexDefinition;
+import org.neo4j.io.fs.FileUtils;
+import org.neo4j.kernel.api.KernelTransaction;
+import org.neo4j.kernel.api.ReadOperations;
+import org.neo4j.kernel.api.Statement;
+import org.neo4j.kernel.api.schema.IndexQuery;
+import org.neo4j.kernel.api.schema.LabelSchemaDescriptor;
+import org.neo4j.kernel.api.schema.index.IndexDescriptor;
+import org.neo4j.kernel.impl.api.KernelStatement;
+import org.neo4j.kernel.impl.core.ThreadToStatementContextBridge;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.storageengine.api.StorageStatement;
+import org.neo4j.storageengine.api.schema.IndexReader;
+import org.neo4j.test.rule.TestDirectory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import static java.util.Arrays.asList;
+
+import static org.neo4j.collection.primitive.PrimitiveLongCollections.count;
+import static org.neo4j.helpers.collection.Iterables.asList;
+import static org.neo4j.test.Unzip.unzip;
+
+public class Start3_2DbOn3_3AndCreateFusionIndexIT
+{
+    private static final String ZIP_FILE = "3_2-db.zip";
+
+    private static final Label LABEL1 = Label.label( "Label1" );
+    private static final Label LABEL2 = Label.label( "Label2" );
+    private static final String KEY1 = "key1";
+    private static final String KEY2 = "key2";
+
+    @Rule
+    public final TestDirectory directory = TestDirectory.testDirectory();
+
+    @Ignore( "Here as reference for how 3.2 db was created" )
+    @Test
+    public void create3_2Database() throws Exception
+    {
+        File storeDir = tempStoreDirectory();
+        GraphDatabaseService db = new GraphDatabaseFactory().newEmbeddedDatabase( storeDir );
+        try
+        {
+            createIndexesAndData( db, LABEL1 );
+        }
+        finally
+        {
+            db.shutdown();
+        }
+        System.out.println( "Db created in " + storeDir.getAbsolutePath() );
```

Comment:
System.out 🎉  (although in ignored test, but anyway)

Reply:
It's intended... you want that when creating the db :)